sandbox/basic.css

```diff
@@ -41,26 +41,29 @@ App > Screen {
   background: $primary-darken-2;
   color: $text-primary-darken-2 ;
   border-right: outer $primary-darken-3;
+  content-align: center middle;
```

Comment:
New rule to align content within a widget.

Reply:
align moves the Widget, content-align moves the text inside a widget. The docs will have nice diagrams to show the difference.

I would expect there to be little reason to adjust content-align for a pre-built widgets. For instance a button will likely always have content-align center middle.

Reply:
It's very subjective, and I'm probably bike-shedding :sweat_smile: , but... wouldn't something like `justify`, `text-justify`, `justify-items` or `justify-content` be more differentiated to `align` than `content-align`? (they all are existing CSS property names)